### PR TITLE
Beta_Integration - add beta tag to integration and PB, remove TPB

### PR DESCRIPTION
**Pull request by bakatzir**
## Status
Ready

## Related Issues
fixes: https://github.com/demisto/etc/issues/18476
fixes: https://github.com/demisto/etc/issues/18498

## Does it break backward compatibility?
   - No

### CODE REVIEW

**Comment by ronykoz** [comment]
How changing the order of which we copy the files fixes anything? 
The Demisto server is down at that point which means it doesn’t matter

**Comment by bakatzir** [comment]
a test playbook was missing sub-playbooks that were in the beta repo so the content was not updated in the server. changing the order we copy meant the needed playbooks were found.

File: Beta_Integrations/Blueliv/Blueliv.yml [hunk 1]
```diff
@@ -48,3 +48,4 @@ script:
   runonce: false
   script: ''
   type: python
+  beta: True
```
Comment (yuvalbenshalom) [inline Beta_Integrations/Blueliv/Blueliv.yml]: replace `True` with `true`

File: Beta_Integrations/MailListener_-_POP3/MailListener_-_POP3.yml [hunk 2]
```diff
@@ -43,12 +43,13 @@ configuration:
...
-display: MailListener - POP3
-name: MailListener - POP3
+display: MailListener - POP3 (Beta)
+name: MailListener - POP3 Beta
 script:
   isfetch: true
   runonce: false
   script: '-'
   type: python
+  beta: True
```
Comment (yuvalbenshalom) [inline Beta_Integrations/MailListener_-_POP3/MailListener_-_POP3.yml]: replace True with true

File: Beta_Integrations/Telegram/Telegram.yml [hunk 3]
```diff
@@ -44,3 +44,4 @@ script:
   runonce: false
   script: '-'
   type: python
+  beta: True
```
Comment (yuvalbenshalom) [inline Beta_Integrations/Telegram/Telegram.yml]: replace True with true

File: Beta_Integrations/playbook-TrendMicro_DDA_DetonateURL.yml [hunk 4]
```diff
@@ -368,5 +368,6 @@ outputs:
 - contextPath: File.Name
   description: File name
   type: string
+beta: True
```
Comment (yuvalbenshalom) [inline Beta_Integrations/playbook-TrendMicro_DDA_DetonateURL.yml]: replace True with true

**Pull request merged**